pilot/pkg/bootstrap/server.go

```diff
@@ -41,13 +41,15 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 
+	mcp "istio.io/api/mcp/v1alpha1"
```

Comment:
for sake of readability, call this mcpapi and the coredatamodel below should be named as mcp